src/spas/libs/member-management/feature/src/lib/member-management/member-management.component.html

```diff
@@ -0,0 +1,2 @@
+<h1>{{title}}</h1>
+<jwadmin-users></jwadmin-users>
```

Comment:
component should be prefixed with ```member-management``` instead of jwadmin.

Reply:
might need to update angular.json and set member-management as the prefix for the member-management lib section

Reply:
OK. Looks like its already updated to prefix member-management in the lib section on line 476 of the angular.json file.